supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-enterprise-linux.adoc

```diff
@@ -2,6 +2,136 @@
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:ansible-playbook[Ansible playbook]
+
+[discrete]
+[[ansible-playbook]]
+==== image:images/yes.png[yes] Ansible playbook (noun)
+*Description*: Playbooks are Ansible’s configuration, deployment, and orchestration language. They can describe a policy you want your remote systems to enforce, or a set of steps in a general IT process. An Ansible playbook is a file that contains one or more Ansible plays.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:ansible-play[Ansible play]
+
+[discrete]
+[[ansible-task]]
+==== image:images/yes.png[yes] Ansible task (noun)
+*Description*: An Ansible play can contain multiple tasks. Ansible tasks are units of action in Ansible. The goal of each task is to execute a module, with very specific arguments.
+An Ansible task is a set of instructions to achieve a state defined, in its broad terms, by a specific Ansible role or module, and fine-tuned by the variables of that role or module.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[apache-web-server]]
+==== image:images/yes.png[yes] Apache web server (noun)
+*Description*: The Apache HTTP Server, colloquially called Apache, is a free and open-source cross-platform web server application, released under the terms of Apache License 2.0. Apache played a key role in the initial growth of the World Wide Web (WWW), and is currently the leading HTTP server. Its process name is `httpd`, which is short for _HTTP daemon_. Red Hat Identity Management (IdM) uses the Apache Web Server to display the IdM Web UI, and to coordinate communication between components, such as the Directory Server and the Certificate Authority (CA).
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:certificate[certificate], xref:certificate-authorities[Certificate Authorities], xref:directory-server[Directory Server]
+
+[discrete]
+[[blueprint]]
+==== image:images/yes.png[yes] blueprint (noun)
+*Description*: Blueprints are simple text files in Tom's Obvious Minimal Language (TOML) format that describe which packages, and what versions, to install into the image. They can also define a limited set of customizations that can be used to build the final image.
+
+*Use it*: yes
+
+*Incorrect forms*: blue print, BluePrint
+
+*See also*:
+
+[discrete]
+[[certificate]]
+==== image:images/yes.png[yes] certificate (noun)
+*Description*: A certificate is an electronic document used to identify an individual, a server, a company, or other entity and to associate that identity with a public key. A certificate provides generally recognized proof of a person's identity. Public-key cryptography uses certificates to address the problem of impersonation.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:certificate-authorities[Certificate Authorities]
+
+[discrete]
+[[certificate-authorities]]
+==== image:images/yes.png[yes] Certificate Authorities (noun)
```

Comment:
I know that these are often abbreviated as "CA", but isn't "certificate authority" lowercase written out?

Please also update the xrefs that link to this if you do lowercase it